.github/workflows/test_examples.yml

```diff
@@ -39,25 +36,27 @@ jobs:
           python -m pip install --upgrade pip
           pip install robotframework==3.1 psutil pyyaml requests
 
-      - name: Test
+      - name: Test person-detection
         run: |
-          cp -r binaries tests
-          renode-test tests/**.robot
+          cd examples/person-detection
+          ./build.sh
+          renode-test person_detection.robot
```

Comment:
If `renode-test` accepts multiple robot files, then having `*.robot` makes sense anyway. I don't know why did we have `**.robot` though